src/main/java/com/joseeduardo/todoist/service/TarefaService.java

```diff
@@ -0,0 +1,11 @@
+package com.joseeduardo.todoist.service;
+
+import com.joseeduardo.todoist.model.CriarTarefaEntradaDTO;
+
+public interface TarefaService {
+	void criar(CriarTarefaEntradaDTO criarTarefaEntradaDTO, Long id);
+
+	void excluirPorID(Long id);
+
+
+}
```

Comment:
Pode deletar os espaços em branco

```suggestion
public interface TarefaService {
	void criar(CriarTarefaEntradaDTO criarTarefaEntradaDTO, Long id);
	void excluirPorID(Long id);
}
```